esp/bindings/http/platform/httpservice.cpp

```diff
@@ -1136,6 +1136,30 @@ EspAuthState CEspHttpServer::authNewSession(EspAuthRequest& authReq, const char*
     return authSucceeded;
 }
 
+void CEspHttpServer::createGetSessionTimeoutResponse(StringBuffer& resp, ESPSerializationFormat format, IPropertyTree* sessionTree)
+{
+    __int64 timeoutAt = sessionTree ? sessionTree->getPropInt64(PropSessionTimeoutAt, 0) : 0;
```

Comment:
Based on my agreement with Miguel, I returns the time stamp for the timeoutAt and the 0 means that the session is timeout. I will add some comment here.